app/controllers/partnerbases_controller.rb

```diff
@@ -0,0 +1,104 @@
+# Provides full CRUD for Partners. These are minimal representations of corresponding Partner records in PartnerBase.
+# Though the functionality of Partners is actually fleshed out in PartnerBase, in DiaperBase, we maintain a collection
+# of which Partners are associated with which Diaperbanks.
+class PartnerbasesController < ApplicationController
+  include Importable
```

Comment:
```suggestion
```

I'am pretty sure we don't need this?

Reply:
deleted!

Reply:
Rebased and ensured all the tests are passing with new routes/language -- ie - "Profile / Application Details"